templates/tilecloudchain-ingress.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.tilecloudchain.enabled -}}
+{{- if .Values.tilecloudchain.ingress.enabled -}}
```

Comment:
would something like this work (untested)?
 `{{- if and .Values.tilecloudchain.enabled .Values.tilecloudchain.ingress.enabled -}}`

Reply:
+1 :-)